docs/modules/getting-started/pages/blue-green.adoc

```diff
@@ -0,0 +1,385 @@
+        </cluster-members>
+    </network>
+    <connection-strategy>
+        <connection-retry>
+            <cluster-connect-timeout-millis>1000</cluster-connect-timeout-millis> <1>
+        </connection-retry>
+    </connection-strategy>
+</hazelcast-client>
+----
+====
+<1> Timeout value in milliseconds for the client to give up to connect to the current cluster.
+For testing/development purposes, set to 1000 milliseconds to see the client connecting to the failover cluster faster than in a production scenario.
+. Create a client failover configuration file and reference the `client-blue` and `client-green` client configurations.
+The name of the client failover configuration file must be `hazelcast-client-failover` (`hazelcast-client-failover.yaml` or `hazelcast-client-failover.xml`).
+Place this failover configuration file in the `resources` directory.
++
+[tabs] 
+==== 
+YAML:: 
++ 
+-- 
+.hazelcast-client-failover.yaml
+[source,yaml]
+----
+hazelcast-client-failover:
+  try-count: 4 <1>
+  clients:
+    - client-blue.yaml
+    - client-green.yaml
+----
+--
+
+XML::
++
+.hazelcast-client-failover.xml
+[source,xml]
+----
+<hazelcast-client-failover>
+    <try-count>4</try-count> <1>
+    <clients>
+        <client>client-blue.xml</client>
+        <client>client-green.xml</client>
+    </clients>
+</hazelcast-client-failover>
+----
+====
+<1> Number of times that the client will try to reconnect to each cluster before shutting down.
++
+In this failover configuration file, you are directing the client to connect to the clusters in the given order from top to bottom;
+see xref:clients:java#ordering-of-clusters-when-clients-try-to-connect[Ordering of Clusters]. So, when you start the client
+(see Step 4 below), it will initially connect to the `blue` cluster. Here is what may happen:
+
+* When the `blue` cluster fails, the client attempts to reconnect to it four times.
+* If the connection is unsuccessful, the client will try to connect to the `green` cluster four times.
+* If these eight connection attempts are unsuccessful, the client shuts down.
+
+== Step 4. Connect the Client to Blue Cluster
+
+In this step, you'll start the client.
+
+[tabs] 
+==== 
+Java:: 
++
+
+. Install the xref:getting-started:install-hazelcast.adoc#use-java[Java client library].
+. Add the following to the `MyClient.java` file.
++
+-- 
+[source,java,subs="attributes+"]
+----
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientFailoverConfig;
+import com.hazelcast.core.HazelcastInstance;
+
+HazelcastInstance client = HazelcastClient.newHazelcastFailoverClient(); <1>
+
+// Following is the example directory structure
+// showing the locations of this Java client code and client/failover configurations.
+//
+//📄 pom.xml
+//📂 src
+//  📂 main
+//    📂 java
+//      📄 MyClient.java
+//    📂 resources
+//      📄 client-blue.yaml
+//      📄 client-green.yaml
+//      📄 hazelcast-client-failover.yaml
+//
```

Comment:
```suggestion
*/
```

We can remove the other `//`. The `/* */` if for multi-line comments, which looks cleaner.